CIPs/0009.md

```diff
@@ -0,0 +1,86 @@
+# CIP [0009]: Standalone signing service for Validators
+
+- Date: 2020-05-14
+- Author: @asaj
+- Status: WIP
+
+## Overview
```

Comment:
Love the idea....

Not sure if `clef` is the answer... but something like `clef`. I would explore the needs of this `remote signer`, i believe clef feature set tries to address a similiar but different problem. And that building a custom remote signer with ingrained feature to do the mutex might be a better impl strategy.

At least, i would do a small research about that as an option.

Reply:
Sorry, I'm not clear why extending `clef` isn't the best option here